apps/shared/avifutil.c

```diff
@@ -121,6 +121,7 @@ static void avifImageDumpInternal(const avifImage * avif, uint32_t gridCols, uin
         }
     }
     printf(" * Progressive    : %s\n", avifProgressiveStateToString(progressiveState));
+    printf(" * CLLI           : %u, %u\n", (uint32_t)avif->clli.maxCLL, (uint32_t)avif->clli.maxPALL);
```

Comment:
If the values are the default/absent/ignore values of (0,0), should this print anything at all, or perhaps some other/additional string indicating that the CLLI is Absent? I'd like to manage enduser expectations a bit here.

Reply:
I agree that (0, 0) should be ignored here.  I've added a conditional to skip the print of the CLLI unless at least one of the clli fields is non-zero.